src/event/model/event_type.rs

```diff
@@ -88,7 +88,7 @@ pub enum EventType {
     SetUuidState,
     #[serde(rename(
         serialize = "UnsupportedNotificationEvent",
-        deserialize = "discussion/restore"
+        deserialize = "discussion/comment/restore"
```

Comment:
This is probably not needed anymore, right?

Reply:
to be honest idk: it was added for compatibility with legacy.